.idea/checkstyle-idea.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="CheckStyle-IDEA">
+    <option name="configuration">
+      <map>
+        <entry key="checkstyle-version" value="8.28" />
```

Comment:
Shouldn't we include the path for active configuration? Something like below, otherwise how does it hook up with our check style plugin? Something like below..

```
<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="CheckStyle-IDEA">
    <option name="configuration">
      <map>
        <entry key="active-configuration" value="LOCAL_FILE:$PROJECT_DIR$/hbase-checkstyle/src/main/resources/hbase/checkstyle.xml:hbase" />
        <entry key="checkstyle-version" value="8.10.1" />
        <entry key="copy-libs" value="false" />
        <entry key="location-0" value="BUNDLED:(bundled):Sun Checks" />
        <entry key="location-1" value="BUNDLED:(bundled):Google Checks" />
        <entry key="location-2" value="LOCAL_FILE:$PROJECT_DIR$/hbase-checkstyle/src/main/resources/hbase/checkstyle.xml:hbase" />
        <entry key="scan-before-checkin" value="false" />
        <entry key="scanscope" value="JavaOnlyWithTests" />
        <entry key="suppress-errors" value="false" />
      </map>
    </option>
  </component>
</project>
```

Reply:
Yes I think we aught... I thought it was included here. I need to review my IDE configs. Will be back.